<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManager</class>
 <widget class="QDialog" name="UserManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../bin/Images/user.png</normaloff>../../../bin/Images/user.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>6</y>
     <width>586</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>User Info</string>
   </property>
   <widget class="QTableWidget" name="tableUserInfo">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>13</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>25</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>18</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>18</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>13</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUserName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAuthority">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>Developor Level1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Developor Level2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Developor Level3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manager Level1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manager Level2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manager Level3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>User Level1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>User Level2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>User Level3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guest</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Authority:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnAddUserInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add User Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReplaceUserInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace User Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteUserInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete User Info</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>376</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel(&amp;C)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok(&amp;O)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
